sim/rogue/between_the_eyes.go

```diff
@@ -0,0 +1,78 @@
+package rogue
+
+import (
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/proto"
+)
+
+func (rogue *Rogue) registerBetweenTheEyes() {
+	if !rogue.HasRune(proto.RogueRune_RuneBetweenTheEyes) {
+		return
+	}
+	flatDamage := rogue.RuneAbilityBaseDamage()
+	comboDamageBonus := rogue.RuneAbilityDamagePerCombo()
+	// TODO: Update damage variance
+	damageVariance := map[int32]float64{
+		25: 20,
+		40: 44,
+		50: 68,
+		60: 108,
+	}[rogue.Level]
```

Comment:
Damage variance is always the same - it's from 0.53 to 0.81 multiplied by `flatDamage + cp * comboDamageBonus`.

Reply:
I updated based on what you said here, but would appreciate a check to make sure I interpreted correctly.